src/devices/Card/Mifare/MifareCard.cs

```diff
@@ -888,6 +913,8 @@ public bool FormatNdef(ReadOnlySpan<byte> keyB = default)
         /// <param name="message">The NDEF Message to write</param>
         /// <param name="writeKeyA">True to write with Key A</param>
         /// <returns>True if success</returns>
+        /// TODO: check the MAD to determine which sectors can be used, instead of assuming
```

Comment:
is the comment for the intellisense comment, meaning the developer needs to check it or it for the code and that needs to be done in a later fix/improvement?

Reply:
It is a note to the developer. I've removed it and will open a bug for the additional change.